tensorflow/python/ops/array_ops.py

```diff
@@ -259,13 +259,6 @@ def rank_internal(input, name=None, optimize=True):
       return gen_array_ops.rank(input, name=name)
 
 
-# DEPRECATED use init_ops.zeros_initializer
-# TODO(irving) Move it to init_ops.py
-def zeros_initializer(shape, dtype=dtypes.float32, partition_info=None):
```

Comment:
Can you leave a forwarder here? This will break downstream code otherwise. Could be as simple as 

```
# DEPRECATED
zeros_initializer = init_ops.zeros_initializer
```